server/src/main/java/org/elasticsearch/bootstrap/BootstrapChecks.java

```diff
@@ -728,13 +727,14 @@ public final BootstrapCheckResult check(BootstrapContext context) {
         }
 
         boolean isAllPermissionGranted() {
-            final SecurityManager sm = System.getSecurityManager();
-            assert sm != null;
-            try {
-                sm.checkPermission(new AllPermission());
-            } catch (final SecurityException e) {
-                return false;
-            }
+            // final SecurityManager sm = System.getSecurityManager();
+            // assert sm != null;
+            // try {
+            // sm.checkPermission(new AllPermission());
+            // } catch (final SecurityException e) {
+            // return false;
+            // }
+            // NOCOMMIT disabled security manager
             return true;
```

Comment:
We can't commit this. But I spent half a day messing with the security manager and gave up. We should be able to fix this.